dist/well-known.js

```diff
@@ -115,6 +116,48 @@ return Promise.all([
       return data;
     });
   }),
+  // FedCM
+  parseResponse('/.well-known/web-identity', r => {
+    return r.json().then(data => {
+        let result = {
+            provider_urls: Array.isArray(data.provider_urls) && data.provider_urls.length > 0 ? data.provider_urls : [],
+            accounts_endpoint: data.accounts_endpoint || null,
+            login_url: data.login_url || null
+        };
+        return result;
+    }).catch(error => {
+        return {
+            error: `Failed to parse JSON: ${error.message}`
+        };
+    });
+  }),
```

Comment:
@max-ostapenko @tunetheweb Thank you for your previous feedback. I was trying to reflect your feedbacks and made the update - not producing an error for invalid "JSON" any more, instead we are responding with default results.
Does this update align better? If this update looks good for you, I will update for the other two parts ('/.well-known/passkey-endpoints' and '/.well-known/webauthn') as well.  

```suggestion
  parseResponse('/.well-known/web-identity', r => {
    return r.text().then(text => {
        let result = {
            provider_urls: [],
            accounts_endpoint: null,
            login_url: null
        };
        try {
            let data = JSON.parse(text);
            result.provider_urls = Array.isArray(data.provider_urls) && data.provider_urls.length > 0 ? data.provider_urls : [];
            result.accounts_endpoint = data.accounts_endpoint || null;
            result.login_url = data.login_url || null;
        } catch (e) {
            // Failed to parse JSON
        }
        return result;
    });
  })
```

Reply:
Looks good.

Among the test websites only https://www.ebay.com/.well-known/passkey-endpoints gives a successful result.
Do you know the examples for the other 2?